src/components/partner/partner.resolver.ts

```diff
@@ -78,7 +78,7 @@ export class PartnerResolver {
     @Parent() partner: Partner,
     @Loader(OrganizationLoader) organizations: LoaderOf<OrganizationLoader>,
   ): Promise<SecuredOrganization> {
-    return await mapSecuredValue(partner.organization, (id) =>
+    return await mapSecuredValue(partner.organization, ({ id }) =>
```

Comment:
Hey @CarsonF - Any advice here? So I need to go deeper and update this? I don't feel like I should have to update this at all, right? So maybe I made a bad change elsewhere. This only fails for neo4j.

Reply:
You changed the DTO but didn't change the cypher queries to reflect the shape change.
https://github.com/SeedCompany/cord-api-v3/blob/c70dcf7e9cf1493b015b4f9c9932a3729579df4e/src/components/partnership/partnership.repository.ts#L160-L162
https://github.com/SeedCompany/cord-api-v3/commit/2a04d882aaa99da9d91958c42cd83111a2b594ba#diff-66096aa79c78355d9c374f6ff9e642df1084d4922af8e42fc2a684ea67eff990

Reply:
Didn't even think about the whole "partnership" aspect of it 🤦

Reply:
Wait, did you mean partnership or partner? @CarsonF

Reply:
Sorry the relevant neo4j repos. partner & org